modules/virt-collecting-data-about-vms.adoc

```diff
@@ -0,0 +1,23 @@
+// Module included in the following assemblies:
+//
+// * virt/logging_events_monitoring/virt-collecting-virt-data.adoc
+
+[id="virt-collecting-data-about-vms_{context}"]
+= Collecting data about virtual machines
+
+Collecting data about malfunctioning virtual machines (VMs) minimizes the time required to analyze and determine the root cause.
+
+.Prerequisites
+
+* Windows VMs:
+** Record the Windows patch update details for Red Hat Support.
+** Install the latest version of the VirtIO drivers. The VirtIO drivers include the QEMU guest agent.
+** If Remote Desktop Protocol (RDP) is enabled, try to connect to the VMs with RDP to determine whether there is a problem with the connection software.
+
+.Procedure
+
+. Collect `must-gather` data for VMs.
+. Collect screenshots of VMs before they start.
```

Comment:
`Collect screenshots of VMs if they have crashed and before they are restarted.`